packages/transformer/src/IModelExporter.ts

```diff
@@ -104,6 +104,11 @@ export abstract class IModelExportHandler {
    */
   public async preExportElement(_element: Element): Promise<void> {}
 
+  /** Look up a target ElementId from the source ElementId.
+   * @returns the target ElementId or [Id64.invalid]($bentley) if a mapping not found.
+   */
+  public findTargetElementId(_sourceElementId: Id64String): Id64String { return Id64.invalid; }
```

Comment:
not sure the `IModelExportHandler` should really be aware of this... the `IModelTransformer` should be...